CharacterCreation/01_01_Races_Human

```diff
@@ -0,0 +1,145 @@
+==============================
+Humans
+==============================
+
+===== Society =====
+
+Humans originated on the planet Earth in the Sol system. They have been exploring space 
+to some degree for almost 800 years, settling their own Terran Empire after the catastrophic 
+events that changed the Earth.
+In general, adult humans range from 5’ to 6’5”. They can be larger or smaller, but individuals
+of those types are generally rare. 
+They are an incredibly prolific and opportunistic species, having completely terraformed and 
+colonized over half a dozen planets and their respective moons. With many more Terran planets 
+on the rise, they already dwarf the small Opalite and Tundari civilizations.
+
+Humans are very culturally dissimilar to each other. Even more so than most other species, 
```

Comment:
Nonblocking: We'll need to indent the paragraphs to get them to display well with the parser.